open-build-service/notqmail.dsc.in

```diff
@@ -0,0 +1,14 @@
+Format: 1.0
+Source: notqmail
+Section: Misc
+Priority: extra
+Binary: notqmail
+Architecture: any
+Version: @version@-@release@
+Maintainer: notqmail <packages@notqmail.org>
+Homepage: https://github.com/notqmail/notqmail
```

Comment:
another notqmail.org

Reply:
This has been addressed in the latest commit #[3eaf213](https://github.com/notqmail/notqmail/pull/76/commits/3eaf2138ce40da0831bfc90852faa8b7c2a2c4ff)